### DISCUSSION TITLE
Load FMU with Femyou not working

### DISCUSSION THREAD

**Discussion by bflo96** [OpenModelica/OpenModelica#12901] [Q&A]
Hello!
I checked and there's a discussion with a similar issue, but it was not marked as solved and the author eventually moved to Python to solve their issue.

I have a program in C#, running on Visual Studio 2019, with target framework **netcoreapp3.1**.
I used Open Modelica v1.22.3 (64-bit).
When I try to load the FMU, it shows this error:

> System.IO.FileNotFoundException: 'Could not find or load the native library: C:\Users\10376\AppData\Local\Temp\Femyou\TestarFemyou.fmu\binaries\win64\TestarFemyou.dll'

I don't understand why it tries to load a native library and why it fails despite the DLL being available when I manually check the folder.
Could it be a compatibility issue? Is there anything I have to do on Open Modelica in terms of specifications of the FMU export to be able to run it?

Thank you!

**Comment by casella** [comment]
Please try with 1.24.0-dev.beta.0. The issue may be already solved.
